packages/mollie-integration/src/mollie-payment-strategy.ts

```diff
@@ -335,8 +318,8 @@ export default class MolliePaymentStrategy implements PaymentStrategy {
         return !!options.form?.fields;
     }
 
-    private _processAdditionalAction(error: any): Promise<InternalCheckoutSelectors> {
-        if (!(error instanceof RequestError) || !some(error.body.errors, {code: 'additional_action_required'})) {
+    private _processAdditionalAction(error: any): Promise<any> {
```

Comment:
```suggestion
    private _processAdditionalAction(error: any): Promise<unknown> {
```
Maybe better to use `unknown` in this case or `void`?